egs2/seame/asr1/run.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+stage=1
+stop_stage=13
+
+train_set="train"
+valid_set="valid"
+test_sets="devman devsge"
+
+asr_config=conf/tuning/train_asr_conformer.yaml
+lm_config=conf/tuning/train_lm_transformer.yaml
+inference_config=conf/decode_asr.yaml
+
+if [ ${stage} -eq 1 ] || [ ! -f "data/train/token.man.2" ]; then
```

Comment:
Maybe just add a check for data/train/token.man.2 here, then through an error is stage is not 1? We would prefer to have the data preprocessing in the main asr.sh

Reply:
Just fixed, please take a look. Thanks!